examples/conv-opt/comparison/gen_conv_models.py

```diff
@@ -0,0 +1,88 @@
+import onnx
+from onnx import numpy_helper, helper, TensorProto
+import numpy as np
+
+# Filter
+sobel = {3: np.array([[1, 0, -1],
+                      [2, 0, -2],
+                      [1, 0, -1]], dtype='float32'),
+         5: np.array([[2, 1, 0, -1, -2],
+                      [3, 2, 0, -2, -3],
+                      [4, 3, 0, -3, -4],
+                      [3, 2, 0, -2, -3],
+                      [2, 1, 0, -1, -2]], dtype='float32'),
+         7: np.array([[3, 2, 1, 0, -1, -2, -3],
+                      [4, 3, 2, 0, -2, -3, -4],
+                      [5, 4, 3, 0, -3, -4, -5],
+                      [6, 5, 4, 0, -4, -5, -6],
+                      [5, 4, 3, 0, -3, -4, -5],
+                      [4, 3, 2, 0, -2, -3, -4],
+                      [3, 2, 1, 0, -1, -2, -3]], dtype='float32'),
+         9: np.array([[4, 3, 2, 1, 0, -1, -2, -3, -4],
+                      [5, 4, 3, 2, 0, -2, -3, -4, -5],
+                      [6, 5, 4, 3, 0, -3, -4, -5, -6],
+                      [7, 6, 5, 4, 0, -4, -5, -6, -7],
+                      [8, 7, 6, 5, 0, -5, -6, -7, -8],
+                      [7, 6, 5, 4, 0, -4, -5, -6, -7],
+                      [6, 5, 4, 3, 0, -3, -4, -5, -6],
+                      [5, 4, 3, 2, 0, -2, -3, -4, -5],
+                      [4, 3, 2, 1, 0, -1, -2, -3, -4]], dtype='float32')
+}
+
+def get_output_shape(i):
```

Comment:
Use two spaces indentation for consistency, just like the onnxruntime-conv2d.py file.